test/functional/ui/messages_spec.lua

```diff
@@ -333,6 +333,38 @@ describe('ui/ext_messages', function()
     ]]}
   end)
 
+  it(':echoerr multiline', function()
+    feed(':lua vim.g.multi = table.concat({ "bork", "fail" }, "\\n")<cr>')
+    feed(':echoerr g:multi<cr>')
+    screen:expect{grid=[[
+      ^                         |
+      {1:~                        }|
+      {1:~                        }|
+      {1:~                        }|
+      {1:~                        }|
+    ]], messages={{
+        content = {{[[bork
+fail]], 2}},
+        kind = "echoerr"
+      }}}
```

Comment:
```suggestion
      }}
    }
```

Reply:
Updated and amended, thanks for the suggestion.